.CI/create_feedstocks.py

```diff
@@ -254,10 +252,24 @@ def write_token(name, token):
                 team.url + "/memberships/" + new_member
             )
 
+    # Update status based on remote
+    subprocess.check_call(['git', 'checkout', os.environ.get('TRAVIS_BRANCH')])
+    subprocess.check_call(['git', 'pull', '--rebase', '--autostash'])
```

Comment:
Though ultimately we abandoned this as we could not keep staged and unstaged things in their properly places. See PR ( https://github.com/conda-forge/staged-recipes/pull/1471 ).